cli/ssh.go

```diff
@@ -424,6 +447,13 @@ func (r *RootCmd) ssh() *clibase.Cmd {
 			FlagShorthand: "l",
 			Value:         clibase.StringOf(&logDirPath),
 		},
+		{
+			Flag:          "remote-forward",
+			Description:   "Enable remote port forwarding.",
```

Comment:
Also provide a description of what `--remote-forward` wants to accept e.g. `remote_port:local_address:local_port`.

Reply:
Done